nipoppy_cli/nipoppy/workflows/base.py

```diff
@@ -69,6 +69,9 @@ def __init__(
         self.logger = logger
         self.dry_run = dry_run
 
+        # for the CLI
+        self.return_code = 0
```

Comment:
random thought

in case we need to start having a growing number of return code,
it may be good to centralize them all in one place for the whole package:

See the ones for BIDS app:

https://bids-standard.github.io/execution-spec/outputs.html#reserved-exit-codes-and-their-definitions

Reply:
Added return codes and colours in `nipoppy.env` module